benefits/enrollment/templates/enrollment/index.html

```diff
@@ -14,7 +14,7 @@
 {% block headline %}
   <div class="col-lg-8">
     <h1 class="pb-lg-8 pb-4">
-      {% translate "Your eligibility is confirmed!<br>You’re nearly there." %}
+      {% translate "We found your record!<br>Now let's enroll your contactless card." %}
```

Comment:
Very minor, but make sure to use the `’` apostrophe character, which was there before in `You’re`